### ISSUE TITLE
Warnings during training, any sentence gets classified as the same intent with the same probability

### ISSUE THREAD

**Issue by jarozajac** [RasaHQ/rasa_nlu#777]
**Rasa NLU version**: 0.10.6

**Used backend / pipeline**: spacy_sklearn, spacy 2.0.5

**Operating system** (windows, osx, ...): windows

**Issue**:
I have a selftrained language model:
-  initialized with word frequencies, word2vec, brown clusters
- trained on a corpus with roughly 20000 sentences 
The model works on its own with some accuracy.

When I create a rasa nlu project based on the tutorial https://nlu.rasa.ai/tutorial.html, I provide a training json file with 2 intents, 1 intent with 18 examples, the other intent with 17 examples, I have 2 issues: 
1) during training I get multiple warnings: sklearn\metrics\classification.py:1135: UndefinedMetricWarning: F-score is ill-defined and being set to 0.0 in labels with no predicted samples.
  'precision', 'predicted', average, warn_for)
2) when running the server, every single input gets classified as the same intent, with the same probability.

Can someone tell me,  what causes this behaviour and how I can fix it?

**Comment by twerkmeister** [comment]
@jarozajac I think, the warning is not important for the problem. It sounds like there is no information to distinguish between the intents. What pipeline are you using?

**Comment by jarozajac** [comment]
@twerkmeister I am using spacy 2.0.5 +sklearn, more precisely
```
"pipeline": [
        "rasa_nlu.utils.spacy_utils.SpacyNLP",
        "rasa_nlu.tokenizers.spacy_tokenizer.SpacyTokenizer",
        "rasa_nlu.featurizers.spacy_featurizer.SpacyFeaturizer",
        "rasa_nlu.featurizers.regex_featurizer.RegexFeaturizer",
        "rasa_nlu.extractors.crf_entity_extractor.CRFEntityExtractor",
        "rasa_nlu.extractors.entity_synonyms.EntitySynonymMapper",
        "rasa_nlu.classifiers.sklearn_intent_classifier.SklearnIntentClassifier"
    ],
```

What information can there be provided? To my knowledge:
- my spacy model provides word vectors, tag maps, brown clusters and word frequencies. It does not have a working lemmatizer and ner, but I doubt that is an issue, the model simply predicts for every word the lemma as the same word and the intents don't use entities yet.
- rasa gets trained on the examples as shown in the tutorial, more exactly the examples are simple 1 to 2 word greeting and agreement examples.

During training the logs show the correct intent types and it looks like spacy loads the correct model... What could I be missing?

**Comment by daysm** [comment]
I'm running into the same issue as @jarozajac when following the restaurant bot tutorial: https://core.rasa.ai/master/tutorial_supervised.html

Same thing happens when using the example from the repo: https://github.com/RasaHQ/rasa_core/tree/master/examples/restaurantbot

Any idea what's going on here?

**Comment by wrathagom** [comment]
@daysm are you using a custom trained spacy 2.0 model? If so please provide more information about how you created the model and what features it has.

If not then you're issue is not related to to this one and instead you should read through #767

**Comment by wrathagom** [comment]
@jarozajac can you provide your custom model and training data (for the rasa piece)?

**Comment by jarozajac** [comment]
I have put the spacy model here: [Spacy model Dropbox](https://www.dropbox.com/s/g36xlctlh74bvpk/sk_sdtbezchyb-0.0.1.tar.gz?dl=0)
The Raas json files are here: [Rasa config Dropbox](https://www.dropbox.com/s/cpckg1yts128ax4/Rasa_NLU.zip?dl=0)
